cirq/protocols/control.py

```diff
@@ -33,7 +33,7 @@ def control(controllee: Union['cirq.Gate', op_tree.OP_TREE],
     """Returns a Controlled version of the given value, if defined.
 
     Controllees define how to be controlled by defining a method
-    controlled_by(self, control_qubits). Note that the method may return
+    control(self, control_qubits). Note that the method may return
```

Comment:
This is incorrect.  ControlledGate.control doesn't have this signature.

I think Gates can no longer support this protocol.

Reply:
Removed Gates from the signature